src/content/docs/kubernetes-pixie/k8s-otel/advance-config.mdx

```diff
@@ -0,0 +1,266 @@
+---
+title: Advanced configuration
+metaDescription: Learn how to configure charts for advanced use cases in Kubernetes with OpenTelemetry.
+tags:
+    - Kubernetes
+    - OpenTelemetry
+    - Advanced Configuration
+    - New Relic Kubernetes integration
+    - New Relic Distribution on OpenTelemetry
+    - NRDOT
+freshnessValidatedDate: never
+---
+
+New Relic provides a set of Helm charts to deploy OpenTelemetry Collector in your Kubernetes cluster. These charts can be customized to suit your specific needs, including advanced configurations for various use cases.
+
+There are several advanced configuration options available for the `nr-k8s-otel-collector` Helm chart. These options allow you to customize the behavior of the OpenTelemetry Collector to better suit your needs. The chart supports both `DaemonSet` and `Deployment` collectors, allowing you to choose the best fit for your use case.
+
+This document provides an overview of some of the key advanced configuration options you can use.
+
+
+## Enable GKE Autopilot or Red Hat OpenShift Compatibility [#enable-provider]
+
+
+To ensure compatibility with specific Kubernetes environments, you can enable provider-specific configurations. This setting ensures compatibility and proper functionality of the OpenTelemetry Collectors by adapting to the specific constraints of these environments. You can enable this option in your [`values.yaml`]() file:
+
+```yaml
+  provider: "GKE_AUTOPILOT" # Or "OPEN_SHIFT" if applicable
+```
+
+
+## Enable Low Data Mode [#low-data]
+
+
+The `LowDataMode` option is enabled by default to ingest only the metrics required by our Kubernetes UIs. This mode reduces the amount of data collected, focusing on essential metrics for Kubernetes monitoring. To fetch additional metrics, you can add new pipelines and configuring the appropriate receivers and processors in your [`values.yaml`]() file using the `extraConfig` section.
```

Comment:
We need to include this section in this doc
https://newrelic.atlassian.net/wiki/spaces/KOT/pages/4408737833/K8s+OTel+docs#How-to-add-new-metrics-in-top-of-LowDataMode?

And also include a link to it in ".... To fetch additional metrics..."